setup.py

```diff
@@ -0,0 +1,34 @@
+from setuptools import setup
+
+
+setup(
+    name="habet",
+    version="0.0.7",
+    author="Tom Osika",
+    author_email="tom.osika@kitware.com",
+    description="The HArmonization BEnchmarking Tool (HABET) streamlines the "
+    "process of trying multiple different harmonization techniques and evaluating "
+    "their performance through a simple command-line interface (CLI)",
+    python_requires=">=3.7",
+    classifiers=[
+        "Programming Language :: Python :: 3",
+        "License :: OSI Approved :: Apache Software License",
+        "Operating System :: OS Independent",
+    ],
+    project_urls={
+        "Bug Tracker": "https://github.com/KitwareMedical/habet/issues"
+    },
+    install_requires=[
+        "dipy==1.5.0",
+        "monai==1.0.1",
+        "neuroCombat==0.2.12",
+        "numpy==1.21.6",
+        "pandas==1.3.5",
+        "pingouin==0.5.2",
+    ],
```

Comment:
I think it's better to replace `==` by `>=`
(assuming we don't actually need functionality that was phased out of the current versions)

In my case when I tried pip installing it ended up downgrading my existing numpy, pandas, etc.

Also, curious, why is monai included?

Reply:
> I think it's better to replace `==` by `>=` (assuming we don't actually need functionality that was phased out of the current versions)
> 
> In my case when I tried pip installing it ended up downgrading my existing numpy, pandas, etc.

Yeah, that was a typo on my part. I just copied the relevant package strings from doing `pip freeze`, forgetting to change the `==` to `>=`.

> Also, curious, why is monai included?
We use monai's `ImageLoader` to read in images with the corresponding affine matrix. We can probably eliminate this dependency, though. Might be something good to tie that in with #10

Reply:
> I just copied the relevant package strings from doing pip freeze, forgetting to change the == to >=.

Ah then you can actually omit the version specification altogether in the `install_requires`, since it's not coming from any knowledge about lower/upper version bounds needed for the package to work.



> We use monai's `ImageLoader` to read in images with the corresponding affine matrix. We can probably eliminate this dependency, though. Might be something good to tie that in with #10

It's also reasonable to include that here in this PR, because monai brings in pytorch which majorly bloats the pip install. Resolving this greatly helps #25 in a practical sense

Reply:
> Ah then you can actually omit the version specification altogether in the `install_requires`, since it's not coming from any knowledge about lower/upper version bounds needed for the package to work.
> 

True, we should pin specific versions at some point. But for now I agree.

> It's also reasonable to include that here in this PR, because monai brings in pytorch which majorly bloats the pip install. Resolving this greatly helps #25 in a practical sense

I agree. Would be good to have some unit tests to make sure that everything works the same after using a different image loader. To put in unit tests, it might be useful to have a default dataset, like in #16. Hmmm.... Not sure how crazy we need to go, but _maybe_ we should do some re-ordering of priorities. Lets discuss this when we meet later.

Reply:
Replaced the monai and dipy dependencies in 4a7f4b9d06c9a364da090263ed8ccf04123cf9f3